_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -2304,6 +2304,7 @@ mfxStatus MfxHwH264Encode::CheckVideoParamQueryLike(
             par.mfx.RateControlMethod != MFX_RATECONTROL_AVBR &&
             par.mfx.RateControlMethod != MFX_RATECONTROL_QVBR &&
             par.mfx.RateControlMethod != MFX_RATECONTROL_VCM &&
+            par.mfx.RateControlMethod != MFX_RATECONTROL_ICQ &&
```

Comment:
So, ICQ mode can be supported by Lowpower mode  too?

Reply:
ICQ mode can't be supported by Lowpower mode now, so I remove this code. @lizhong1008